resources/user-agents/browsers/yabrowser/yabrowser-1-20-14-12.json

```diff
@@ -269,6 +269,68 @@
           ]
         }
       ]
+    },
+    {
+      "userAgent": "Yandex Browser #MAJORVER#.#MINORVER# for iOS",
```

Comment:
I can pull more history of yabrowser to see what the version range is. I added the iOS versions here because I did run into a version older than 15.

In the iOS App Store, the oldest version they show is 15.12, but that usually doesn't list every version that was ever available (that version was released on January 13th, 2016).

I tried to use the Internet Archive to look at the App Store page and see what was the oldest version it had indexed is, but it doesn't have that page indexed (either didn't index it, or it has changed URLs over time):

App store URL: https://itunes.apple.com/us/app/yandex-browser-fast-and-secure/id483693909?mt=8
Internet Archive of that: https://web.archive.org/web/*/https://itunes.apple.com/us/app/yandex-browser-fast-and-secure/id483693909?mt=8